content/docs/tina-cloud/beta-migration.mdx

```diff
@@ -1,15 +1,19 @@
 ---
 title: Upgrading from Alpha to Beta
+next: ''
+previous: ''
 ---
 
+> Tina is currently in full release. While the content of this page may still be useful, we recommend using later versions of Tina.
```

Comment:
Should this be in a warning callout?

Reply:
That UX could be improved!
Did you make a suggestion to the TinaCMS team?

Reply:
Yep thinking the same thing 👍 

Just had a chat to @Ben0189  about it, and since they're using a library it might not be easy to change. It looks like they're using [Plate](https://platejs.org/) for it which does have some kind of horizontal scroll option.

Also discovered the options are accessible (see below), if you try and click a half-obscured toolbar button.

I'll make a PBI and let the PO (Matt?) know.

![Screenshot 2024-09-12 at 12 54 39 pm](https://github.com/user-attachments/assets/d95ee15c-4d8f-4de7-9c39-60fac3e1f981)